webpack/assets/javascripts/react_app/common/helpers.js

```diff
@@ -51,6 +51,24 @@ export const clearSpaces = string => string.trim().replace(/\s\s+/, ' ');
 export const getDisplayName = Component =>
   Component.displayName || Component.name || 'Component';
 
+export const urlBuilder = (controller, action, id = undefined) =>
+  `/${controller}/${id ? `${id}/` : ''}${action}`;
+
+export const urlWithSearch = (base, searchQuery) =>
+  `/${base}?search=${searchQuery}`;
+
+export const stringIsInteger = value => {
+  // checking for positive integers only
+  const reg = new RegExp('^[0-9]+$');
+  return reg.test(value);
+};
+
+export const KEY_CODES = {
+  TAB_KEY: 9,
+  ENTER_KEY: 13,
+  ESCAPE_KEY: 27,
```

Comment:
@Laviro we already have it defined somewhere right?

Reply:
We have it inside the `AutoComplete` component, but we can use this helpers now.

Reply:
IMHO we should have a `keyCodes.js` file.

Reply:
Added keyCodes.js